### PR TITLE
Search subcmd test case

### PR DESCRIPTION
**Pull request by proditis**

### CODE REVIEW

**Comment by g0rchy** [comment]
nice addition

**Review by g0rchy — APPROVED** [review]
nice add

File: orunmila.go [hunk 1]
```diff
@@ -30,6 +29,9 @@ func main() {
 		fmt.Fprintln(flag.CommandLine.Output(), "  describe Set the database description")
 		fmt.Fprintln(flag.CommandLine.Output(), "  vacuum   Rebuild the database file, repacking it into a minimal amount of disk space")
 	}
+	exitCode := 0
+	var err error
+	err = nil
```
Comment (proditis) [inline orunmila.go:34]: not needed here

**Review by proditis — COMMENTED** [review]
LGTM

**Pull request merged**